src/http/commands/help/presenter.rb

```diff
@@ -70,28 +70,23 @@ def initialize(manifest)
               self.manifest = manifest
             end
 
-            def render_html_list(data, skip_wrapper: false)
+            def render_html_list(data)
               html = ""
-
               case data
               when ::Hash
-                html << "<ul>" unless skip_wrapper
+                html << "<ul>"
                 data.each do |key, value|
                   html << "<li>#{key}"
-                  html << "<ul>"
-                  html << render_html_list(value, skip_wrapper: true)
-                  html << "</ul>"
+                  html << render_html_list(value)
                   html << "</li>"
                 end
-                html << "</ul>" unless skip_wrapper
+                html << "</ul>"
               when ::Array
-                html << "<ul>" unless skip_wrapper
```

Comment:
I think that because we are rendering a collection here, we need the `<ul>` and a `<li>` for each child.